dev-python/logbook/logbook-1.5.3.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_IN_SOURCE_BUILD=1
+DISTUTILS_USE_SETUPTOOLS=bdepend
+
+inherit distutils-r1
+
+DESCRIPTION="A logging replacement for Python"
+HOMEPAGE="https://logbook.readthedocs.io/en/stable/
+	https://github.com/getlogbook/logbook
+	https://pypi.org/project/Logbook/"
+SRC_URI="https://github.com/getlogbook/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+distutils_enable_tests pytest
+distutils_enable_sphinx docs
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? ( app-arch/brotli[${PYTHON_USEDEP},python] )
+	doc? ( >=dev-python/sphinx-1.1.3-r3[${PYTHON_USEDEP}] )"
+RDEPEND="dev-python/redis-py[${PYTHON_USEDEP}]"
+
+PATCHES=( "${FILESDIR}"/${PN}-0.4.2-objectsinv.patch )
```

Comment:
Do you still need this patch? If it's for intersphinx, d_e_s should handle it.

Reply:
Removed the patch, it works without it